alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -200,7 +203,15 @@ pub fn new(config: &Options, window_size: WindowSize) -> Option<Pty> {
 
     let cmdline = win32_string(&cmdline(config));
     let cwd = config.working_directory.as_ref().map(win32_string);
-
+    let mut creation_flags = EXTENDED_STARTUPINFO_PRESENT;
+    let custom_env_block = convert_custom_env(&config.env);
+    let custom_env_block_pointer = match &custom_env_block {
+        Some(custom_env_block) => {
+            creation_flags |= CREATE_UNICODE_ENVIRONMENT;
+            custom_env_block.as_ptr() as *mut std::ffi::c_void
+        },
+        None => ptr::null_mut(),
+    };
```

Comment:
Please don't just put everything in massive blocks with no documentation again. This could is much more readable with sensible spacing and a comment or two. Otherwise nobody will know what is going on at a glance.

Reply:
Sorry, quite hard to come from another world with another style requirements.

I've added a small comment (which is also dangerous in this repo, given how many review notes were for those, but let's see) and a new linebreak to separate the whole arguments' preparation block out.